docs/news.rst

```diff
@@ -3,6 +3,108 @@
 Release notes
 =============
 
+.. _release-2.9.0:
+
+Scrapy 2.9.0 (YYYY-MM-DD)
+-------------------------
+
+Highlights:
+
+-   Per-domain request settings.
+-   Compatibility with new cryptography_ and new parsel_.
+-   TBD
+
+New features
+~~~~~~~~~~~~
+
+-   Settings correponding to :setting:`DOWNLOAD_DELAY`,
+    :setting:`CONCURRENT_REQUESTS_PER_DOMAIN` and
+    :setting:`RANDOMIZE_DOWNLOAD_DELAY` can now be set on a per domain basis
+    via the new :setting:`DOWNLOAD_SLOTS` setting. (:issue:`5328`)
+
+-   Added :func:`scrapy.utils.request.request_to_curl`, a function to produce a
+    curl command from a :class:`~scrapy.Request` object. (:issue:`5892`)
+
+-   Values of :setting:`FILES_STORE` and :setting:`IMAGES_STORE` can now be
+    :class:`pathlib.Path` instances. (:issue:`5801`)
+
+-   :func:`scrapy.utils.request.request_from_curl` now supports $-prefixed
```

Comment:
I am unsure about `LinkExtractor` and `get_func_args`, but all other list items starting with this one feel to me like bug fixes rather than new features.